packages/protocol/test/governance/validators.ts

```diff
@@ -2181,39 +2182,15 @@ contract('Validators', (accounts: string[]) => {
       await registerValidator(validator)
       await registerValidatorGroup(group)
       await validators.affiliate(group)
+      await mockLockedGold.addSlasher(accounts[3])
     })
 
-    describe('when the sender is one of three approved contract addresses', () => {
-      beforeEach(async () => {
-        await registry.setAddressFor(CeloContractName.DowntimeSlasher, validator)
-        await registry.setAddressFor(CeloContractName.DoubleSigningSlasher, accounts[3])
-        await registry.setAddressFor(CeloContractName.GovernanceSlasher, accounts[5])
-        await registry.setAddressFor(CeloContractName.Governance, accounts[4])
-      })
-
-      it('should succeed when the sender is the downtime slasher contract', async () => {
-        await validators.forceDeaffiliateIfValidator(validator)
-        const parsedValidator = parseValidatorParams(await validators.getValidator(validator))
-        assert.equal(parsedValidator.affiliation, NULL_ADDRESS)
-      })
-
-      it('should succeed when the sender is the double signing slasher contract', async () => {
+    describe('when the sender is one of the whitelisted slashing addresses', () => {
+      it('should succeed', async () => {
         await validators.forceDeaffiliateIfValidator(validator, { from: accounts[3] })
         const parsedValidator = parseValidatorParams(await validators.getValidator(validator))
         assert.equal(parsedValidator.affiliation, NULL_ADDRESS)
       })
-
-      it('should succeed when the sender is the governance contract', async () => {
-        await validators.forceDeaffiliateIfValidator(validator, { from: accounts[4] })
-        const parsedValidator = parseValidatorParams(await validators.getValidator(validator))
-        assert.equal(parsedValidator.affiliation, NULL_ADDRESS)
-      })
-    })
-
-    describe('when the sender is not an approved address', () => {
```

Comment:
Why remove this?

Reply:
Change from https://github.com/celo-org/celo-monorepo/pull/2347